Makefile

```diff
@@ -48,6 +48,12 @@ linkcheck:
 install:
 	pip install -e .[development]
 
+.PHONY: install-hooks
+install-hooks:
+	@git config --local core.hooksPath .git-hooks/
+	@chmod +x .git-hooks/*
+	@echo "Git hooks installed."
```

Comment:
What are the `@`s for?

Reply:
It runs the command silently, without the command itself being printed. It's useful in front of the `echo` command, otherwise you have 
```
@echo "Git hooks installed."
"Git hooks installed."
```
on the screen, which looks a bit silly. The usage in front of `git` and `chmod` command is debatable, one may argue that obfuscates what is happening.

Reply:
Well TIL! I'm cool with keeping them silent.